test/test_arraycontext.py

```diff
@@ -249,7 +253,7 @@ def get_real(ary):
         return ary.real
 
     def get_imag(ary):
-        return ary.real
+        return ary.real  # FIXME?
```

Comment:
Explain?

Reply:
It seemed strange to me that a `get_imag` function returns the real part, but I wasn't sure if this is intentional or not.

Reply:
Changed to return the imag part.